tests/integration/pipeline.ts

```diff
@@ -429,6 +429,7 @@ describe('Pipeline Editor tests', () => {
   it('kfp pipeline should display custom components', () => {
     cy.createPipeline({ type: 'kfp' });
     cy.openPalette();
+    cy.get('.palette-flyout-category[value="Kubeflow Pipelines Nodes"]');
```

Comment:
We need to add a similar selector in https://github.com/elyra-ai/elyra/blob/de163f3d1db186d4b4a45b8d60d6018aeb585140/tests/integration/pipeline.ts#L456

`cy.get('[value="Apache Airflow Nodes"]').click();`